css/home.css

```diff
@@ -0,0 +1,455 @@
+}
+
+.sidebar .sidebar-content a{
+	color: #8F816C;
+}
+
+
+.sidebar .sidebar-content p {
+	margin: 0;
+}
+
+.sidebar .sidebar-content ul {
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+}
+
+.sidebar .sidebar-content ul li {
+	margin: 1em 0;
+}
+
+.sidebar .sidebar-content ul li:before {
+	display: inline-block; 
+	background: url('../images/sprites.png') no-repeat; 
+	overflow: hidden; 
+	content: "";
+	margin-right: 1em;
+	background-position: -234px -263px; width: 12px; height: 13px;
+}
+
+#main-footer {
+	font-size: 0.8em;
+	background-color: #C9BBA9;
+	color: #8F816C;
+	padding: 1em;
+}
+
+#main-footer a {
+	padding: 0 0.5em;
+	margin: 0;
+	color: #8F816C;
+	border-right: 1px dashed #8F816C;
+}
+
+#main-footer a:first-child {
+	border-left: 1px dashed #8F816C;
+}
+
+
+@media all and (min-width: 992px) {
+	/* Wrapper
+	-----------------------------------------------------------------------------*/
+	#wrapper {
+		width: 850px;
+		margin: 0 auto;
+	}
+
+	/* Header
+	-----------------------------------------------------------------------------*/
+	#main-header {
+		text-align: center;
+	}
+
+	.sprite-menu {
+		display: none;
+	}
+
+	#logo {
+		float: left;
+	}
+
+	#logo-banner {
+		text-align: left;
+		padding-left: 2em;
+		margin: 0 auto; 
+		display: inline-block;
+	}
+
+	#search {
+		float: right;
+	}
+
+	/* Main menu
+	-----------------------------------------------------------------------------*/
+	#main-menu {
+		display: block !important;
+		width: 100%;
+		opacity: 1 !important;
+		background-color: transparent;
+		position: relative;
+		top: 0;
+		margin-top: 2em;
+	}
+
+	#main-menu ul {
+		display: table;
+		width: 100%;
+	}
+
+	#main-menu ul li {
+		border-right: 1px dotted #EAE2D7;
+		display: table-cell;
+		border-radius: 0;
+		padding: 1em;
+		margin: 0;
+		cursor: pointer;
+	}
+
+	#main-menu ul li.first {
+		border-radius: 8px 0 0 8px;
+	}
+
+	#main-menu ul li.last {
+		border-radius: 0 8px 8px 0;
+	}
+
+	/* Middle
+	-----------------------------------------------------------------------------*/
+	.middle {
+		width: 100%;
+		position: relative;
+	}
+
+	.container {
+		width: 100%;
+		float: left;
+		overflow: hidden;
+	}
+
+	.content {
+		padding: 0 290px 0 0;
+	}
+
+	.sprite-bottle-small {
+		background-position: -0px -0px; width: 250px; height: 250px;
+	}
+
+	#news {
+		border-radius: 8px;
+		overflow: hidden;
+		margin: 1em 0;
+	}
+
+	#news section {
+		float: left;
+		width: 50%;
+		border-radius: 0;
+		overflow: auto;
+		margin: 0;
+	}
+
+	/* Sidebar
+	-----------------------------------------------------------------------------*/
+	.sidebar {
+		float: left;
+		width: 270px;
+		margin-left: -270px;
+		position: relative;
+	}
+
+	/* Footer
+	-----------------------------------------------------------------------------*/
+	#main-footer {
+		padding: 1.5em;
+	}
+
+	#footer-wrapper {
+		width: 850px;
+		padding-top: 1em;
+		margin: auto;
+	}
+
+	#main-footer p {
+		float: left;
+		margin: 0;
+	}
+
+	#main-footer nav {
```

Comment:
Not all groups of links on a page need to be in a nav element — the element is primarily intended for sections that consist of major navigation blocks. In particular, it is common for footers to have a short list of links to various pages of a site, such as the terms of service, the home page, and a copyright page. The footer element alone is sufficient for such cases; while a nav element can be used in such cases, it is usually unnecessary.